crashbin_app/templates/crashbin_app/home.html

```diff
@@ -3,11 +3,119 @@
+                        <div class="card-body">
+                            <p class="card-text">All bins</p>
+                            <h3 class="card-title">{{ bins.count }}</h3>
+                        </div>
+                    </div>
+                    <div class="text-center bg-light">
+                        <div class="card-body">
+                            <p class="card-text">All reports</p>
+                            <h3 class="card-title">1000</h3>
+                        </div>
+                    </div>
+                    <div class="text-center bg-light">
+                        <div class="card-body">
+                            <p class="card-text">Bins you maintain</p>
+                            <h3 class="card-title">1000</h3>
+                        </div>
+                    </div>
+                    <div class="text-center bg-light">
+                        <div class="card-body">
+                            <p class="card-text">Reports you maintain</p>
+                            <h3 class="card-title">1000</h3>
+                        </div>
+                    </div>
+                    <div class="text-center bg-light">
+                        <div class="card-body">
+                            <p class="card-text">Bins you subscribed</p>
+                            <h3 class="card-title">1000</h3>
+                        </div>
+                    </div>
+                    <div class="text-center bg-light">
+                        <div class="card-body">
+                            <p class="card-text">Reports you subscribed</p>
+                            <h3 class="card-title">1000</h3>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col px-4 pb-3 bg-white">
+                <h1 class="my-2">Maintaining</h1>
+                {% for bin in bins %}
+                    <div class="mb-2">
+                        <div id="bin-header-{{ forloop.counter0 }}" class="list-group-header bg-light">
+                            <button class="btn btn-block d-flex flex-wrap p-2" data-toggle="collapse"
+                                    data-target="#bin-collapse-{{ forloop.counter0 }}"
+                                    aria-expanded="true" aria-controls="bin-collapse-{{ forloop.counter0 }}">
+                                <h4 class="text-justify p-2 my-auto">{{ bin.name }}</h4>
+                                <div class="d-flex flex-row p-2 ml-auto my-auto">
+                                    <div class="d-flex flex-row-reverse flex-wrap">
+                                        {% for label in bin.labels.all %}
+                                            <span class="badge text-wrap m-1 p-2"
+                                                  style="background-color: {{ label.color }}; color: white">{{ label.name }}</span>
```

Comment:
We should probably select the text color based on the background color (at least at some later stage) - when you have a light-colored or even white label, the text would be unreadable currently.